FTBFS with ECM version >= 5.80.0.

Error message
```
../src/lib/configlib/iso639.cpp: In constructor 'fcitx::kcm::Iso639::Iso639()':
../src/lib/configlib/iso639.cpp:55:41: error: 'ISOCODES_ISO639_2_JSON' was not declared in this scope
   55 |     iso639_2data_ = readAlpha3ToNameMap(ISOCODES_ISO639_2_JSON, "639-2");
      |                                         ^~~~~~~~~~~~~~~~~~~~~~
../src/lib/configlib/iso639.cpp:56:41: error: 'ISOCODES_ISO639_3_JSON' was not declared in this scope
   56 |     iso639_3data_ = readAlpha3ToNameMap(ISOCODES_ISO639_3_JSON, "639-3");
      |                                         ^~~~~~~~~~~~~~~~~~~~~~
../src/lib/configlib/iso639.cpp:57:41: error: 'ISOCODES_ISO639_5_JSON' was not declared in this scope
   57 |     iso639_5data_ = readAlpha3ToNameMap(ISOCODES_ISO639_5_JSON, "639-5");
      |                                         ^~~~~~~~~~~~~~~~~~~~~~
```
with a `config.h` with those lines:
```
/* #undef ISOCODES_ISO639_2_JSON */
/* #undef ISOCODES_ISO639_3_JSON */
/* #undef ISOCODES_ISO639_5_JSON */
```
----
This should be due to ECM version >= 5.80.0 ships `/usr/share/ECM/find-modules/FindIsoCodes.cmake` that overrides your `cmake/FindIsoCodes.cmake`.

--- a/pkgs/tools/inputmethods/fcitx5/fcitx5-configtool.nix
+++ b/pkgs/tools/inputmethods/fcitx5/fcitx5-configtool.nix
@@ -19,13 +19,13 @@
 
 mkDerivation rec {
   pname = "fcitx5-configtool";
-  version = "5.0.2";
+  version = "5.0.4";
 
   src = fetchFromGitHub {
     owner = "fcitx";
     repo = "fcitx5-configtool";
     rev = version;
-    sha256 = "sha256-kw0KIbS5SVMf6kR/9xsYiChHXQBM0enSVXyh0QfiiPY=";
+    sha256 = "sha256-UO3Ob+bFQ/2Vqb8YpD9tfmfZt5YLUyoqcbtsHLaVOzE=";
   };
 
   cmakeFlags = [

